Tools/glib/run-webkit-app.py

```diff
@@ -37,11 +37,19 @@ def get_default_configuration(output_dir):
         return None
 
 
+def prepend_path_var(env, name, value):
+    if name in env:
+        env[name] = ':'.join((value, env[name]))
```

Comment:
Likely I would have written this instead with f-string interpolation:

```suggestion
        env[name] = f'{value}:{env[name]}'
```

But definitely not a blocker, your option works equally well.